app/src/main/java/org/dosomething/letsdothis/ui/fragments/HubFragment.java

```diff
@@ -92,31 +92,39 @@ public void onAttach(Activity activity)
     public void onStop()
     {
         super.onStop();
-        //FIXME I don't think this is the right place for this
         EventBusExt.getDefault().unregister(this);
     }
 
     @Override
-    public void onActivityCreated(@Nullable Bundle savedInstanceState)
+    public void onResume()
     {
-        super.onActivityCreated(savedInstanceState);
-        RecyclerView recyclerView = (RecyclerView) getView().findViewById(R.id.recycler);
-
-        //FIXME get real user
-        User user = new User(null, "firstName", "lastName", "birthday");
-        user.id = AppPrefs.getInstance(getActivity()).getCurrentUserId();
+        EventBusExt.getDefault().register(this);
```

Comment:
FIXME: Crash here if you leave this screen and come back. "Subscriber class org.dosomething.letsdothis.ui.fragments.HubFragment already registered"